premake/app.lua

```diff
@@ -12,7 +12,7 @@ function app( name )
 	debugdir 'src/%{prj.name}/Assets'
 	links( libraries )
 	links( third_party_libraries )
-	location 'build/%{_ACTION}'
+	location ''
```

Comment:
Can just not set the location at all, or at least put `'.'` if you wanna be explicit 🙂